tests/testthat/test_PlainTextParam.R

```diff
@@ -56,21 +94,38 @@ test_that("storeResults,loadResults,PlainTextParam,MsExperiment works", {
     ## Loading data again
     load_mse <- loadResults(object = MsExperiment(), param)
     expect_true(inherits(load_mse, "MsExperiment"))
-    expect_equal(sampleData(mse), sampleData(load_mse))
-    a <- spectra(mse)
+    expect_equal(sampleData(tmp), sampleData(load_mse))
```

Comment:
Can I ask why you create a `tmp`  object here ? is it because otherwise we permanently change the `mse` (or `xmse_full`) object for the later tests ?

Reply:
I think `testthat` changes the variables within each unit test locally, not globally - it's more that when I evaluate the unit tests line by line I change the variables in my R session and then I get unexpected errors later. I generally avoid overwriting the "globally" defined variables where possible. Also, I specifically use a variable name `tmp` to remember that I will overwrite this variable for sure later...

Reply:
oki fair thanks !